recipes/r-dotcall64/meta.yaml

```diff
@@ -27,10 +27,11 @@ requirements:
     - r-base
     - posix                # [win]
     - {{native}}toolchain  # [win]
-    - gcc                  # [not win]
+    - gcc                  # [linux]
```

Comment:
Ah... This is a Fortran "extension," so we are probably better of with `gcc`. I don't really like the `yum_requirements.txt` solution. It is cumbersome and I don't really see any advantage.